march_gazebo_plugins/src/com_controller.cpp

```diff
@@ -0,0 +1,210 @@
+#include <gazebo/common/common.hh>
+#include <gazebo/physics/physics.hh>
+#include <march_shared_resources/GaitActionGoal.h>
+#include <ros/callback_queue.h>
+#include <ros/ros.h>
+
+namespace gazebo
+{
+class ComController : public ModelPlugin
+{
+public:
+  void Load(physics::ModelPtr _parent, sdf::ElementPtr /*_sdf*/)
```

Comment:
You should provide the `override` keyword to indicate that it has been overriden from `ModelPlugin`
```suggestion
  void Load(physics::ModelPtr _parent, sdf::ElementPtr /*_sdf*/) override
```